functions/go/publish-functions.sh

```diff
@@ -25,13 +25,14 @@ image_tag=${TAG:-latest}
 prefix="release-go-functions-"
 [[ "${image_tag}" = "${prefix}"* ]] && image_tag="${image_tag#$prefix}"
 
+make
```

Comment:
This should probably the either way around, Makefile has a command for also building and publisher docker image. It can execute helper bash scripts as implementation detail. The GH workflows can then invoke make for everything.

This can be done in a future PR.

Reply:
@mengqiy @Shell32-Natsu Probably want to add a name to the TODO or track all these issues in a bug. Anti pattern would be adding a TODOs that are not addressed.

Reply:
Ping. Who's following up on this? Is that a follow up PR?

Reply:
filed https://github.com/GoogleContainerTools/kpt/issues/1251